scripts/spack/devtools_configs/blueos_3_ppc64le_ib_p9/packages.yaml

```diff
@@ -15,121 +15,121 @@
-  # the following per spack docs
   netlib-lapack:
-    buildable: false
     externals:
-    - spec: netlib-lapack@3.6.1
-      prefix: /usr
+      # This is the one that has the trailing underscores in name mangling, needed for MFEM compat
+      - spec: netlib-lapack@3.9.0
+        prefix: /usr/tcetmp/packages/lapack/lapack-3.9.0-gcc-7.3.1/lib/
+    buildable: false
 
-# System level packages to not build
+  # System level packages to not build
   autoconf:
-    buildable: false
+    buildable: false 
     externals:
-    - spec: autoconf
-      prefix: /usr
+      - spec: autoconf@2.69
+        prefix: /usr
   automake:
-    buildable: false
+    buildable: false 
     externals:
-    - spec: automake
-      prefix: /usr
+      - spec: automake@1.13.4
+        prefix: /usr
   binutils:
     buildable: false
     externals:
-    - spec: binutils
+    - spec: binutils@2.27
       prefix: /usr
   bzip2:
     buildable: false
     externals:
-    - spec: bzip2
+    - spec: bzip2@1.0.6
       prefix: /usr
   diffutils:
     buildable: false
     externals:
-    - spec: diffutils
+    - spec: diffutils@3.3
       prefix: /usr
   epoxy:
     buildable: false
     externals:
-    - spec: epoxy
+    - spec: epoxy@0.0.0
       prefix: /usr
   findutils:
     buildable: false
     externals:
-    - spec: findutils
+    - spec: findutils@4.5.11
       prefix: /usr
   gettext:
-    buildable: false
+    buildable: false 
     externals:
-    - spec: gettext
-      prefix: /usr
+      - spec: gettext@0.19.8.1
+        prefix: /usr
   ghostscript:
     buildable: false
     externals:
-    - spec: ghostscript
+    - spec: ghostscript@9.25
       prefix: /usr
   groff:
     buildable: false
     externals:
-    - spec: groff
+    - spec: groff@1.22.2
       prefix: /usr
   libtool:
     buildable: false
     externals:
-    - spec: libtool
-      prefix: /usr
-  libuuid:
-    buildable: false
-    externals:
-    - spec: libuuid
+    - spec: libtool@2.4.2
       prefix: /usr
   m4:
     buildable: false
     externals:
-    - spec: m4
+    - spec: m4@1.4.16
       prefix: /usr
   perl:
     buildable: false
     externals:
-    - spec: perl
+    - spec: perl@5.16.3
       prefix: /usr
   pkg-config:
     buildable: false
     externals:
-    - spec: pkg-config
+    - spec: pkg-config@0.27.1
       prefix: /usr
   readline:
     buildable: false
     externals:
-    - spec: readline
+    - spec: readline@6.2
       prefix: /usr
   tar:
-    buildable: false
     externals:
-    - spec: tar
-      prefix: /usr
+      - spec: tar@1.26
+        prefix: /usr
+    buildable: false
 
-# Globally lock in version of devtools
+  # Lock in versions of Devtools
   cmake:
-    version: [3.14.5]
-    buildable: false
     externals:
-    - spec: cmake
-      prefix: /usr/tce/packages/cmake/cmake-3.14.5
+    - spec: cmake@3.18.0
+      prefix: /usr/tce/packages/cmake/cmake-3.18.0
+    buildable: false
```

Comment:
I'm curious: are just the devtool packages being built with cmake 3.18?

I see in the blueos host-config the header is still
`# CMake executable path: /usr/tce/packages/cmake/cmake-3.14.5/bin/cmake`

Reply:
Yes this is only for the devtools.  Due to not wanting to rebuild the devtools everytime (it more than doubles the builds), we have to split the configs like this.  I really wish there was a better way but I don't know of one.